test/address.js

```diff
@@ -545,6 +551,11 @@ describe('Address', function() {
       address.toString().should.equal('2N7T3TAetJrSCruQ39aNrJvYLhG1LJosujf');
     });
 
+    it('can create an address from a set of public keys with a nested witness program', function() {
+      var address = Address.createMultisig(publics, 2, Networks.livenet, true);
+      address.toString().should.equal('3PpK1bBqUmPK3Q6QPSUK7BQSZ1DMWL6aes');
```

Comment:
is this going to work? Isn't networks.livenet a particl network?

Reply:
Probably not, i fixed the errors preventing the tests from running to completion. But i haven't gone through the 1000 odd tests that fail to fix them. That will take a bit more time